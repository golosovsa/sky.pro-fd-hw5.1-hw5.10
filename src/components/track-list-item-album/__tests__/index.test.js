import ShallowRenderer from 'react-test-renderer/shallow'
import TrackListItemAlbum from '..'

const TrackListItemAlbumWrapped = ({ children }) => {
  return <TrackListItemAlbum>{children}</TrackListItemAlbum>
}

describe('<TrackListItemAlbum />', () => {
  describe('Snapshot test', () => {
    it('should render successfully and match snapshot', () => {
      const renderer = new ShallowRenderer()
      const tree = renderer.render(<TrackListItemAlbumWrapped>children</TrackListItemAlbumWrapped>)

      expect(tree).toMatchSnapshot()
    })
  })
})
