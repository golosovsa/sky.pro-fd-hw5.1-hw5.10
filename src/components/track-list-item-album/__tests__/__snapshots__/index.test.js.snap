// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrackListItemAlbum /> Snapshot test should render successfully and match snapshot 1`] = `
<styled.p>
  children
</styled.p>
`;
